docs/schema/clientwebpartproperty-complextype-apphostwebfeatures.md

```diff
@@ -126,7 +125,7 @@ If the schema defines specific requirements, such as **sequence**, **minOccurs**
 <td align="left"><p>Values of the xs:boolean type.</p></td>
 </tr>
 <tr class="odd">
-<td align="left"><p>**Name**</p></td>
+<td align="left"><p>Name</p></td>
```

Comment:
The repo guidance is to turn any HTML table you touch into a Markdown table.  Use the online convertor tool to make this process easy, but you'll need to fine tune the output.